src/UI/words.tsx

```diff
@@ -568,8 +570,14 @@ const dict = {
 
   /** Discovered Resources related text */
   "discovered.column.resource_id": "Resource Id",
+  "discovered.column.agent": "Agent",
+  "discovered.column.value": "Value",
+  "discovered.column.type": "Type",
   "discovered.column.managed_resource": "Managed resource",
   "discovered.column.discovery_resource": "Discovery resource",
+  "discovered.column.show_details": "Show Details",
+  "discovered.column.disabled_actions":
+    "This resource has no managed or discovery resource associated.",
```

Comment:
This one is not used afaics ?

Reply:
Good catch, I had it initially, but wasn't needed since the Kebab menu from patternfly is not able to display disabled tooltips